Shopify/cli
# Unify output token systems

### WHY are these changes introduced?

Starts us on the journey to a single system for output. Allows us to output raw strings or tokenized text, using the ui-kit system rather than the old `output` system of tokens.

Future PRs will start porting over functionality. Raw strings can just be moved right over, while those which include tokens will have to be migrated more manually.



### WHAT is this pull request doing?



For raw strings and TokenItems, proxies `outputInfo` and `outputDebug` to a new `renderToken` function. This is implicitly tested by all our tests which check outputs.

### How to test your changes?



Run different commands on the CLI, and make sure everything looks OK. Use verbose and non-verbose mode.

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
